const createError = require('http-errors');

exports.getAll = (req, res, next) => {
  res.send('respond with a resource');
};
